backend/core/src/units/entities/unit.entity.ts

```diff
@@ -156,6 +160,27 @@ class Unit {
   @IsOptional({ groups: [ValidationsGroupsEnum.default] })
   @IsBoolean({ groups: [ValidationsGroupsEnum.default] })
   bmrProgramChart?: boolean | null
+
+  @ManyToOne(() => UnitType, { eager: true, nullable: true })
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @ValidateNested({ groups: [ValidationsGroupsEnum.default] })
+  @Type(() => UnitType)
+  unitTypeRef?: UnitType | null
+
+  @ManyToOne(() => UnitRentType, { eager: true, nullable: true })
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @ValidateNested({ groups: [ValidationsGroupsEnum.default] })
+  @Type(() => UnitRentType)
+  unitRentType?: UnitRentType | null
+
+  @ManyToOne(() => UnitAccessibilityPriorityType, { eager: true, nullable: true })
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @ValidateNested({ groups: [ValidationsGroupsEnum.default] })
+  @Type(() => UnitAccessibilityPriorityType)
+  unitAccessibilityPriorityType?: UnitAccessibilityPriorityType | null
```

Comment:
I think these have similar issues to the handling of unit type. For example, isn't unitAccessibilityPriorityType replacing priorityType, or rather, shouldn't it?